config/index.js

```diff
@@ -493,6 +493,57 @@ var conf = convict({
       default: '.+@mozilla\\.com$',
       env: 'LASTACCESSTIME_UPDATES_EMAIL_ADDRESSES'
     }
+  },
+  signinUnblock: {
+    codeLength: {
+      doc: 'Number of base36 digits to make up an unblockCode',
+      default: 8,
+      env: 'SIGNIN_UNBLOCK_CODE_LENGTH'
+    },
+    codeLifetime: {
+      doc: 'How long an unblockCode should be valid for',
+      format: 'duration',
+      default: '1 hour',
+      env: 'SIGNIN_UNBLOCK_CODE_LIFETIME'
+    },
+    enabled: {
+      default: true
+    },
+    allowEmailRegex: {
+      doc: 'If feature enabled, allow sign-in unblock for email addresses matching this regex.',
+      format: Array,
```

Comment:
Can we ditch the `Array` part of this?

I already have a plan to come back and swap out `signinConfirmation.forceEmailRegex` to `RegExp` if/when my convict PR is accepted. I don't really foresee a use-case when an array of regexes is required as a result of the single regex becoming too long or unwieldy. In reality this is pretty much always going to just be for enabling features for Mozilla employees, imho.